elk/training/classifier.py

```diff
@@ -0,0 +1,64 @@
+from torch.nn.functional import (
+    binary_cross_entropy_with_logits as bce_with_logits,
+    cross_entropy,
+)
+from typing import Optional
+import torch
+
+
+class Classifier(torch.nn.Module):
+    """Linear classifier trained with supervised learning."""
+
+    def __init__(
+        self, input_dim: int, num_classes: int = 1, device: Optional[str] = None
+    ):
+        super().__init__()
+
+        self.linear = torch.nn.Linear(input_dim, num_classes, device=device)
+        self.linear.bias.data.zero_()
+        self.linear.weight.data.zero_()
```

Comment:
this made me slightly worried at first, but I guess it's always fine since its output is never used as input to another layer?

Reply:
Oh I was doing this because I think logistic regression models are usually initialized to zero? It _shouldn't_ matter since this is always a convex problem, although maybe in weird cases it could matter due to the 1e-4 stopping condition